### PR TITLE
Merging the latest master changes into the spet_release branch.

### PR DESCRIPTION
**Pull request by raj-prince**
### Description
- Syncing latest changes of master branch to sept_release branch.
- Resolved conflicts in multiple files.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Did basic sanity for all the basic file system operations.
2. Unit tests - automation.
3. Integration tests - Ran successfully, automation. Although integration test failing intermittenly, but not triggered due to this change. That is also happening in perf test of master branch.

### CODE REVIEW

**Review by vadlakondaswetha — COMMENTED** [review]
too many changes. Please run integration tests on this PR and update in the description.

File: DEBIAN/copyright [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: gcsfuse
+Upstream-Contact: gcs-fuse-maintainers@google.com
+
+Files: *
+Copyright: Copyright 2020 Google Inc.
+License: Apache-2.0
+
+License: Apache-2.0
```
Comment (gargnitingoogle) [inline DEBIAN/copyright:9]: This line is duplicate. Is it supposed to be this way?

Comment (ashmeenkaur) [reply DEBIAN/copyright:9]: Yes this is in accordance to https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#stand-alone-license-stanza

Comment (gargnitingoogle) [reply DEBIAN/copyright:9]: ok.

**Review by sethiay — APPROVED** [review]

**Pull request merged**